baldrick/plugins/tests/test_github_pull_requests.py

```diff
@@ -21,7 +21,7 @@ def setup_module(module):
 
 
 def teardown_module(module):
-    PULL_REQUEST_CHECKS[:] = module.PULL_REQUEST_CHECKS_ORIG[:]
+    PULL_REQUEST_CHECKS = module.PULL_REQUEST_CHECKS_ORIG.copy()
```

Comment:
Oh, whoops, this is borked

Reply:
Did it? Not obvious from Travis log.

Reply:
Oh right, it looks like the tests actually passed, but flake doesn't like it